internal/common/testing/container/container.go

```diff
@@ -45,20 +45,28 @@ type Containers struct {
 type Container struct {
 	ID           ID
 	waitForPorts []uint16
-	ports        nat.PortMap
+	cont         *types.ContainerJSON
 	t            *testing.T
 }
 
-// AddrForPort returns the endpoint with host:port for the container port.
-func (c *Container) AddrForPort(port uint16) string {
-	for dockerPort, bindings := range c.ports {
+// AddrForPort returns the host and port for the container port.
+func (c *Container) AddrForPort(port uint16) (string, uint) {
```

Comment:
Please split this change in a separate PR (everything in internal/common)

Reply:
Removed it as it isn't very essential.